### ISSUE TITLE
icarus wings gliding instead of flying

### ISSUE THREAD

**Issue by jasssw** [CammiePone/Icarus#123]
hi all, i have an issue with the wings. The wings are suddenly acting like an elytra when i double click the space bar, and i cant seem to be able to fly. ive checked the settings and everything but im not sure what is wrong. can anyone help?

**Comment by UpcraftLP** [comment]
Hey o/
Wings have always acted like an elytra. Except that you can accelerate while you have food - did you run out of food maybe?

If not, could you perhaps share what Minecraft version + mod version you are using, and which modpack, if any?

**Comment by jawkee** [comment]
Hi I am also having this issue! I am using 1.20.1 Java Forge, and mod versions Icarus-NeoForge-2.10.0.jar. I did just now notice it says NeoForge. I use curseforge for my modpack I play wtih friends, and it registers it as compatible with my version.

More information on the issue: I can place them in my back slot, I can "glide" like as if I'm using an eltrya without rockets, but I can not "fly" like using an elytra with rockets. There is no way to get into the sky. No amount of jumping (or trying to use rockets) will get me to "fly" off.

**Comment by UpcraftLP** [comment]
yes, in 1.20.1 forge and neoforge are compatible with each other, that works just fine.

Flying works by simply holding down W, but you cannot fly straight up. try it at an angle.
If that doesnt work, make sure your hunger bar is filled sufficiently, and if you are wearing any armor, perhaps take that off or change the config to make it not slow you down.

If that doesnt fix the issue, please provide your full mod list so I can try and reproduce the issue.

**Comment by jawkee** [comment]
Thank you for your reply! It seems using the armor from the mod Ice and Fire makes it impossible to use the wings with the armor slow feature on. Wearing that armor doesn't just "slow" the flight, but makes it impossible to get into the air at all.  I went ahead and turned the armor slows feature off to make the wings more available for use. Thanks for your help!
